src/js/page/odoo/commands/common/renew_database.mjs

```diff
@@ -0,0 +1,74 @@
+// @flow strict
+// Copyright Baptiste <swano@ik.me>
+// License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+// $FlowIgnore
+import i18n from 'i18next';
+import rpcQuery from '@odoo/rpc';
+import {ARG} from '@trash/constants';
+import type {CMDCallbackArgs, CMDCallbackContext, CMDDef} from '@trash/interpreter';
+import type Terminal from '@odoo/terminal';
+
+type SystemParameter = {
+  key: string,
+  value: string,
+};
+
+async function cmdRenewDatabase(this: Terminal, kwargs: CMDCallbackArgs, ctx: CMDCallbackContext) {
+  const uuid = crypto.randomUUID();
+  const now = new Date();
+  const createDate = now.toISOString().slice(0, 19).replace('T', ' ');
+  const expirationDate = new Date(now);
+  expirationDate.setDate(now.getDate() + (kwargs.expires_in || 30));
```

Comment:
I think this isn't working as expected... and you can manage "default" values in the command definition :)